### ISSUE TITLE
Chiseled marble

### ISSUE THREAD

**Issue by xJon** [MrTJP/ProjectRed#1495]
**Describe the bug**
When I try to chisel marble the output options include gem blocks from ProjectRed.

**To Reproduce**
1. Open game
2. Get chisel & marble block
3. Chisel the marble block to any block of any gem you want
4. Score!

**Expected behavior**
To be honest I didn't expect that to happen.

**Versions**
Include versions of the dependencies. And since you're looking at it, make sure ProjectRed is up to date. Don't submit bugs for old versions, as they could have been fixed.
* CCL Version: 1.12.2-3.2.2.353
* FMP Version: 1.12.2-2.6.1.81
* MrTJPCore Version: 1.12.2-2.1.4.43
* ProjectRed Version: 1.12.2-4.9.3.116
* Chisel Version: 1.12.2-0.2.1.35
* UniDict Version: 1.12.2-2.9.2

**Screenshots/Videos**
![Screenshot_2](https://user-images.githubusercontent.com/8945347/61184020-eee28b80-a68b-11e9-937f-720bc02b8079.png)

**Comment by MrTJP** [comment]
Wow. Chisel probably expects all variations of the same block are the same except with different styles. That’s a very dumb assumption to make. I’ll look into it.  If you could open this ticket in that mod’s GitHub, maybe the dev can give some insight.

**Comment by xJon** [comment]
👍

**Comment by HaddockA** [comment]
> 
> 
> Wow. Chisel probably expects all variations of the same block are the same except with different styles. That’s a very dumb assumption to make. I’ll look into it. If you could open this ticket in that mod’s GitHub, maybe the dev can give some insight.

I posted an issue earlier than the OP, an issue that affects more than just one mod and I provided all the information required, yet you completely ignored my post? Why?

**Comment by MrTJP** [comment]
Figured out that this issue was 100% my fault due to some very old Chisel integration that I didn't know we had. It has been resolved.

**Comment by tterrag1098** [comment]
Great! Glad to see someone properly sending both item and block too, the less guesswork chisel has to do the better ;)

**Comment by ccfb3ee765a58cae** [comment]
@MrTJP I think this change may have accidentally made the ProjectRed Compat module require ProjectRed Exploration to be installed.

After replacing these versions

    ProjectRed-1.12.2-4.9.2.110-Base.jar    ProjectRed-1.12.2-4.9.2.110-fabrication.jar  ProjectRed-1.12.2-4.9.2.110-lighting.jar
    ProjectRed-1.12.2-4.9.2.110-compat.jar  ProjectRed-1.12.2-4.9.2.110-integration.jar  ProjectRed-1.12.2-4.9.2.110-mechanical.jar

With these versions (9f6264b0fe611a211c00b62a1b47f457924bcf5e, to be specific)

    ProjectRed-1.12.2-4.9.3.1-Base.jar    ProjectRed-1.12.2-4.9.3.1-fabrication.jar  ProjectRed-1.12.2-4.9.3.1-lighting.jar
    ProjectRed-1.12.2-4.9.3.1-compat.jar  ProjectRed-1.12.2-4.9.3.1-integration.jar  ProjectRed-1.12.2-4.9.3.1-mechanical.jar

I get

```
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from projectred-compat (projectred-compat)
Caused by: java.lang.NoClassDefFoundError: mrtjp/projectred/exploration/BlockDecorativeStone
	at mrtjp.projectred.compatibility.Services$.<init>(Services.scala:15)
	at mrtjp.projectred.compatibility.Services$.<clinit>(Services.scala)
	at mrtjp.projectred.compatibility.CompatibilityProxy_server.preinit(proxies.scala:10)
	at mrtjp.projectred.ProjectRedCompatibility$.preInit(ProjectRedCompatibility.scala:13)
	at mrtjp.projectred.ProjectRedCompatibility.preInit(ProjectRedCompatibility.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraftforge.fml.common.FMLModContainer.handleModStateEvent(FMLModContainer.java:637)
	at sun.reflect.GeneratedMethodAccessor9.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:219)
	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:197)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:136)
	at net.minecraftforge.fml.common.Loader.preinitializeMods(Loader.java:627)
	at net.minecraftforge.fml.client.FMLClientHandler.beginMinecraftLoading(FMLClientHandler.java:252)
	at net.minecraft.client.Minecraft.func_71384_a(Minecraft.java:467)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:378)
	at net.minecraft.client.main.Main.main(SourceFile:123)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.ClassNotFoundException: mrtjp.projectred.exploration.BlockDecorativeStone
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 53 more
Caused by: java.lang.NullPointerException
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:182)
	... 55 more
```

**Comment by MrTJP** [comment]
Yup, it is some weird compiler issue. Apparently something I did creates a hard reference, even though obviously I added nothing that should cause that.

**Comment by MrTJP** [comment]
I've addressed the dependency bug in 4a8a2b9b62eb71dcd3d44825769b644874df5e67.
